types/schemas/postSubmissionApplication/data/Appeal.ts

```diff
@@ -0,0 +1,55 @@
+import {Date} from '../../../shared/utils';
+import {PrimaryApplicationType} from '../../prototypeApplication/enums/ApplicationType';
+import {AppealDecision} from '../enums/AppealDecision';
+
+type AppealBase = {
+  /**
+   * Reason for appeal
```

Comment:
nit: If you prefix this with `@description Reason for appeal` it'll get encoded in generated schema doc ! See available keywords here https://json-schema.org/understanding-json-schema/reference/annotations

(Also totally understandable/relatable if these are still intended as notes-to-self at this stage :sweat_smile:)

Reply:
thats handy to know! Currently we don't know that much about the appeals data so this may be a free text field - we'll have to wait and see for this one! 😅